packages/ui/src/utils/DataFilterUtil.ts

```diff
@@ -54,5 +54,16 @@ export const DataFilterUtil = {
     }
 
     return wallets
+  },
+
+  deduplicateExplorerListingsFromConnectors(listings: Listing[]) {
```

Comment:
NB: Understand this is for #804 but was still confused with this function. Could you please expand.

Reply:
If wagmi connector exists who's name matches entry from our explorer api, connector will remain displayed while explorer listing gets hidden. This name also gave me a headache :D Open to suggestions here tbf